### ISSUE TITLE
Modularise project

### ISSUE THREAD

**Issue by hendersontrent** [hendersontrent/ScratchNeuralNetwork.jl#3]
Probably should have separate scripts/subfolders for:

- Activation functions
- Activation function derivatives
- Loss functions
- Neural Network calls (feedforward, backprop, function to train the network)
